### ISSUE TITLE
Cannot export lights

### ISSUE THREAD

**Issue by fire** [KhronosGroup/glTF-Blender-IO#786]
**Describe the bug**

Blender 2.81 cannot export punctual lights with the splash art scene.

**To Reproduce**

1. Export file as glb
1. Selected punctual lights and cameras

**Expected behavior**

Glb file should be created.

**Screenshots**

![Screenshot from 2019-11-22 09-08-30](https://user-images.githubusercontent.com/32321/69445686-b0c73980-0d07-11ea-830a-d08b46a8bdd4.png)

**.blend file/ .gltf**
[Blender 2 Truncated.zip](https://github.com/KhronosGroup/glTF-Blender-IO/files/3880624/Blender.2.Truncated.zip)

https://cloud.blender.org/p/gallery/5dd6d7044441651fa3decb56

**Version**
 - OS:  Fedora 31 Linux
 - Blender Version: 2.81

**Comment by julienduroure** [comment]
Confirmed.
Bug is when we are trying to export "area" lights
(default scene --> select light --> change type from "point" to "area" --> export --> bug)

**Comment by fire** [comment]
Is an area light just geometry with emission?

**Comment by julienduroure** [comment]
@fire An area light is directional, where an emissive plane is multidirectional.

--> We are going to ignore these type of light for now

**Comment by asarra** [comment]
*push*
This bug still exists!

**Comment by donmccurdy** [comment]
@asarra This is not a bug — not all of Blender's light types exist in glTF or other software. Only point, spot, and directional lights can be exported to glTF 2.0 currently. If you're looking for something else to be added to glTF, I would recommend commenting on https://github.com/KhronosGroup/glTF/issues/1940.

**Comment by asarra** [comment]
Thanks and sorry for my ignorant  comment. I didn't know that this was the case. I'm new to this blender and unity stuff.
May I ask if there's an equivalent thing that can be done to give an area light effect (want to add "led stripes into the sides of a room)?

**Comment by donmccurdy** [comment]
Are you trying to export a model to bring into any particular software? The answer might depend a little on that, but I'd probably suggest using an emissive effect (in recent versions of Blender, there's an Emissive socket on the Principled BSDF material). You may need to enable a bloom post-processing effect to get a good visual result from the emissive material — this would be true in Blender's Eevee renderer, and in realtime renderers like three.js and Unity. Baked lighting can also give some nice effects with a bit more work.

**Comment by asarra** [comment]
Hey again.
I was trying to import it in unity and baking it. It didn't work as I've expected it. It didn't look the same like in Blender.
Could not get my head around lights in general in unity, thus I've switched to three js now.

I've replaced the area lights with 4 directional lights and made the walls a bit shorter so that the sun lights can go through the small gaps. Not what I really wanted either, but I do not have another choice for now, I suppose.

I'll look into that bloom post-processing.
